cmake/Modules/ObsHelpers_macOS.cmake

```diff
@@ -360,27 +360,17 @@ function(setup_obs_bundle target)
       POST_BUILD
       COMMAND
         /bin/sh -c
-        "plutil -replace SUFeedURL -string https://obsproject.com/osx_update/stable/updates_${CMAKE_OSX_ARCHITECTURES}.xml \"$<TARGET_BUNDLE_CONTENT_DIR:${target}>/Info.plist\""
+        "plutil -replace SUFeedURL -string https://obsproject.com/osx_update/stable/updates_v2.xml \"$<TARGET_BUNDLE_CONTENT_DIR:${target}>/Info.plist\""
       VERBATIM)
 
     add_custom_command(
       TARGET ${target}
       POST_BUILD
       COMMAND
         /bin/sh -c
-        "plutil -replace SUPublicDSAKeyFile -string OBSPublicDSAKey.pem \"$<TARGET_BUNDLE_CONTENT_DIR:${target}>/Info.plist\""
+        "plutil -replace SUPublicEDKey -string \"<INSERT ED KEY HERE>\" \"$<TARGET_BUNDLE_CONTENT_DIR:${target}>/Info.plist\""
```

Comment:
This either needs the public key to be added, or we can make it configurable and then add it as an env var/secret to CI.

Reply:
Add it as cache variables, like `-DSPARKLE_APPCAST_URL` and `-DSPARKLE_KEY_FILE`.

The _actual_ fix would be to have a separate `Info.plist.in` just for the app bundle, add the keys there, with both of these cache variables as their values, and just have CMake generate the file automatically.

If Sparkle is not enabled or those cache keys are not set, the plist will just contain empty values, which is fine by me.

Reply:
For ED keys we actually can't use a key file, but have to put the key itself into the property. Same CMake changes apply though.